FrontEnd/src/components/ProfilePage/ProfilePageComponents/ProfileContent.js

```diff
@@ -52,13 +57,44 @@ const ProfileContent = (props) => {
         <div className={css['content__container']}>
             <div className={css['content']}>
                 <div className={css['info-links-profile']}>
-                    {INFOLINKS.map((element) => (
+                {INFOLINKS.map((element) => {
+                    const isLinkEnabled =
+                        (props.profile.is_registered && props.profile.is_startup) ||
+                        (props.profile.is_registered && element.title !== 'Стартап') ||
+                        (props.profile.is_startup && element.title !== 'Інформація про товари/ послуги' && element.title !== 'Додаткова інформація');
+                    return isLinkEnabled ? (
                         <NavLink
                             className={({ isActive }) => (`${css['infolink']} ${isActive && css['infolink__active']}`)}
                             to={`/profile${element.link}`}
                             key={element.title}
-                        >{element.title}</NavLink>
-                    ))}
+                        >
+                            {element.title}
+                        </NavLink>
+                    ) : (
+                        <Tooltip
+                            key={element.title}
+                            placement="bottomLeft"
+                            title={element.tooltipText}
+                            overlayInnerStyle={{
+                                display: 'flex',
+                                borderRadius: '2px',
+                                background: 'var(--main-grey-90, #25292C)',
+                                color: 'var(--main-white, #FFF)',
+                                fontFeatureSettings: 'calt',
+                                fontFamily: 'Inter',
+                                letterSpacing: '-0.14px',
+                            }}
```

Comment:
Is it possible to move these styles to the CSS?

Reply:
I've customized here styles of antd Tooltip component. According to the documentation styles of the tooltip inner content could be changed with overlayInnerStyle property, which type is an object. I didn't find the way to move that styles to the CSS and apply then.